content/metadata.yaml

```diff
@@ -1,26 +1,48 @@
 ---
-title: "Manuscript Title"
+title: "Prediction of cancer mutation states across multiple data modalities reveals the utility and redundancy of gene expression and DNA methylation"
 keywords:
-  - markdown
-  - publishing
-  - manubot
+  - machine learning
+  - functional genomics
+  - gene expression
+  - epigenetics
+  - cancer
+  - pan-cancer
 lang: en-US
 authors:
-  - github: johndoe
-    name: John Doe
-    initials: JD
-    orcid: XXXX-XXXX-XXXX-XXXX
-    twitter: johndoe
-    email: john.doe@something.com
+  - github: jjc2718
+    name: Jake Crawford
+    initials: JC
+    orcid: 0000-0001-6207-0782
+    twitter: jjc2718
+    email: jcra@pennmedicine.upenn.edu
     affiliations:
-      - Department of Something, University of Whatever
+      - Genomics and Computational Biology Graduate Group, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA
+      - Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA, USA
     funders:
       - Grant XXXXXXXX
-  - github: janeroe
-    name: Jane Roe
-    initials: JR
-    orcid: XXXX-XXXX-XXXX-XXXX
-    email: jane.roe@whatever.edu
+  - name: Brock C. Christensen
+    initials: BCC
+    orcid: 0000-0003-3022-426X
     affiliations:
-      - Department of Something, University of Whatever
-      - Department of Whatever, University of Something
+      - Department of Epidemiology, Geisel School of Medicine, Dartmouth College, Lebanon, NH, USA
+    funders:
+      - Grant XXXXXXXX
+  - github: mchikina
+    name: Maria Chikina
+    initials: MC
+    orcid: 0000-0003-2550-5403
+    affiliations:
+      - Department of Computational and Systems Biology, School of Medicine, University of Pittsburgh, Pittsburgh, PA, USA
+    funders:
+      - Grant XXXXXXXX
+  - github: cgreene
+    name: Casey S. Greene
+    initials: CSG
+    orcid: 0000-0001-8713-9213
+    twitter: GreeneScientist
+    email: casey@greenelab.com
+    affiliations:
+      - Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO, USA
+      - Center for Health AI, University of Colorado School of Medicine, Aurora, CO, USA
+    funders:
+      - Grant XXXXXXXX
```

Comment:
```suggestion
      - National Institutes of Health's National Cancer Institute (R01 CA237170)
      - National Institutes of Health's National Human Genome Research Institute (R01 HG010067)
--


<br class="Apple-interchange-newline" style="caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none;">
```

Reply:
Did you want this bit of HTML in the last line here? It seems to break Manubot so I'm going to get rid of it for now, but let me know if it was intentional.